src/Lykke.Service.PayInternal/Modules/ServiceModule.cs

```diff
@@ -114,6 +124,9 @@ private void RegisterServiceClients(ContainerBuilder builder)
             builder.RegisterType<LykkeMarketProfile>()
                 .As<ILykkeMarketProfile>()
                 .WithParameter("baseUri", new Uri(_settings.CurrentValue.MarketProfileServiceClient.ServiceUrl));
+            var ninjaServiceClient = new QBitNinjaClient(_settings.CurrentValue.NinjaServiceClient.ServiceUrl);
```

Comment:
Better not to construct object yourself. Delegate object construction to autofac and just register the type.